lib/zinnia.js

```diff
@@ -365,8 +365,22 @@ export async function run ({
     )
     childProcesses.push(Object.assign(childProcess, { moduleName: module }))
 
+    const resetTimeout = () => {
+      if (timeoutId) clearTimeout(timeoutId)
+      timeoutId = setTimeout(() => {
+        onActivity({
+          type: 'error',
+          message: `${capitalize(module)} has been inactive for 5 minutes, restarting...`
+        })
+        controller.abort()
+      }, 5 * 60 * 1000)
+    }
+    let timeoutId
```

Comment:
Would you mind moving this line before `resetTimeout` so that `resetTimeout` does not use a variable that was defined yet (from the POV of a reader)?